db/load-data.mjs

```diff
@@ -0,0 +1,84 @@
+import postgres from 'postgres';
+import { generateUsers } from "./generate-data.mjs";
+
+async function makeInsertQuery(sql, data) {
+  const columns = Object.keys(data);
+  const values = columns.map(column => data[column]);
+  const columnsList = columns.join(', ');
+  const placeholders = columns.map((_, i) => `$${i + 1}`).join(', ');
+  
+  // Using raw string for column names (they're safe as they come from our code)
+  // and parameterized values for the actual data
+  return await sql.unsafe(`
```

Comment:
[nit] This is fine, but was there an issue with running columnList and placeholders through the sql function like `sql(columnNames)`?